src/Map/TileCover.cs

```diff
@@ -49,7 +76,15 @@ public static class TileCover {
 		/// <param name="coord"> Geographic coordinate. </param>
 		/// <param name="zoom"> Zoom level. </param>
 		/// <returns>The to tile identifier.</returns>
-		public static UnwrappedTileId CoordinateToTileId(Vector2d coord, int zoom) {
+		/// <example>
+		/// Convert a geocoordinate to a TileId:
+		/// <code>
+		/// var unwrappedTileId = TileCover.CoordinateToTileId(new Vector2d(40.015, -105.2705), 18);
+		/// Debug.Log("UnwrappedTileId: " + unwrappedTileId.ToString());
```

Comment:
`Debug.Log` is Unity specific right? 
Should we make this not depend on unity. `Console.write` should work here in both Unity and Non-unity cases?